intelmq/bots/experts/uwhoisd/expert.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import socket
+
+from intelmq.lib.bot import Bot
+
+
+class UniversalWhoisExpertBot(Bot):
+    """ Universal Whois expert bot get the whois entry related an a domain,
+    hostname, IP address, or ASN from a centralised uWhoisd instance """
+    server: str = 'localhost'
+    port: int = 4243
+
+    def _whois(self, query: str) -> str:
+        print(query)
+        bytes_whois = b''
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
+            sock.connect((self.server, self.port))
+            sock.sendall('{}\n'.format(query).encode())
```

Comment:
I have exciting news for you: We dropped Python < 3.6 support, so we can use format strings =)

Reply:
nice!